### DISCUSSION TITLE
Welcome to karamelo Discussions!

### DISCUSSION THREAD

**Discussion by adevaucorbeil** [adevaucorbeil/karamelo#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by karligor** [comment]
Hello, everyone!
I'm trying to install and compile Karamelo on Windows, following the instructions in the text file. However, it doesn't seem to be working properly.
Is there an updated version of the installation procedure available?

**Comment by adevaucorbeil** [comment]
Sorry, there is no support for windows to date.
